// Copyright (c) 2020 forkbase

syntax = "proto3";

package distrkvs;

service Store {
  // Provides a value for each key request
  rpc Get (GetRequest) returns (GetResponse) {}
  rpc Put (PutRequest) returns (PutResponse) {}
}

message GetRequest {
  string key = 1;
}

message GetResponse {
  string value = 1;
}

message PutRequest {
  string key = 1;
  string value = 2;
}

message PutResponse {
  string value = 1;
}
